.github/workflows/push.yml

```diff
@@ -0,0 +1,38 @@
+name: DualGovernance push tests
```

Comment:
For push actions, I suggest running both unit and integration tests. These tests always run on a freshly deployed setup and cover the full functionality of the Dual Governance system.